src/components/hv-option/types.ts

```diff
@@ -8,14 +6,14 @@
  *
  */
 
-export type State = {|
-  refreshing: boolean,
-|};
+export type State = {
+  pressed: boolean;
```

Comment:
Is this right?

Reply:
I think the diff is messed up, see the file names

Reply:
You're right. That is an odd choice GitHub made to mark this as a refactor.